project_event/models/project_task.py

```diff
@@ -123,6 +123,18 @@ def create(self, vals):
             elif vals['activity_task_type'] == 'task':
                 vals['code'] = self.env['ir.sequence'] \
                     .next_by_code('project.task.task')
+        return_create = super(Task, self).create(vals)
+        if vals['activity_task_type'] == 'activity':
+            vals['parent_id'] = return_create.id
+            vals['message_follower_ids'] = None
+            self.create_main_task(vals)
+        return return_create
+
+    @api.multi
+    def create_main_task(self, vals):
```

Comment:
@eilst: def create_main_task(self, vals) method funtional logic  is the same  replicate condition of  elif case in the def create(self, vals) methods .

Reply:
done

Reply:
@eilst the new task is displayed in the activity list of the event

Reply:
done